LocalPayment/src/main/java/com/braintreepayments/api/LocalPaymentClient.java

```diff
@@ -126,6 +130,10 @@ public void onResult(@Nullable Configuration configuration, @Nullable Exception
                             @Override
                             public void onResult(@Nullable LocalPaymentResult localPaymentResult, @Nullable Exception error) {
                                 if (localPaymentResult != null) {
+                                    String pairingID = localPaymentResult.getPaymentId();
```

Comment:
nit, I think on android we use `Id` instead of `ID`, honestly this is one of the hardest things for me to remember switching between iOS and Android. 🙈

Reply:
Indeed
Addressed on [edf7015](https://github.com/braintree/braintree_android/pull/934/commits/edf7015b48c8682da98e44de3c1485ea1a7320a8)